bosh-deployment.yml

```diff
@@ -224,6 +225,18 @@ instance_groups:
         signups_enabled: false
         invitations_enabled: false
 
+  - name: ntpd
```

Comment:
Was this intended as ntpd, or ntp_server?

Reply:
This is intended, the repo has ntp_server, but the bosh.io release has ntpd